README.es-ES.md

```
@@ -0,0 +1,283 @@
+     ...
+   }
+   ```
+
+---
+
+## Inicio Rápido
+
+### Requisitos Previos
+
+- Go 1.23.2 o posterior
+- **Un endpoint RPC dedicado de Solana** (vea [Configuración RPC](#️-important-rpc-endpoint-setup) arriba - ¡esto es crucial!)
```

markdownlint-cli2 (0.23.1)

[warning] Link fragments should be valid

(MD051, link-fragments)